steps/assets/aws/config.tf

```diff
@@ -1 +0,0 @@
-../../../config.tf
```

Comment:
Why can't we keep the symlink?

Reply:
> Why can't we keep the symlink?

We might be able to, but we don't need it for `openshift-install` (only one step :), and I wasn't sure how well `http.FileSystem` handled symlinks.